src/Draco.Compiler/Internal/Documentation/Extractors/XmlDocumentationExtractor.cs

```diff
@@ -0,0 +1,106 @@
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using Draco.Compiler.Internal.Symbols;
+using Draco.Compiler.Internal.Symbols.Metadata;
+using Draco.Compiler.Internal.Symbols.Synthetized;
+
+namespace Draco.Compiler.Internal.Documentation.Extractors;
+
+/// <summary>
+/// Extracts XML into <see cref="SymbolDocumentation"/>.
+/// </summary>
+internal sealed class XmlDocumentationExtractor
```

Comment:
IIRC this class was a lot messier the last time I saw it, I love the simplification!